src/SmartComponents/Group/add-group-modal.js

```diff
@@ -20,7 +20,7 @@ const AddGroupModal = ({
   updateGroup
 }) => {
   const onSubmit = data => {
-    data.user_ids = selectedUsers;
+    data.user_list = selectedUsers.map((user) => { return { username: user };});
```

Comment:
You should never override function params instead copy it into new variable.
If you want to return object directly you can sue this notation instead
```javascript
const something = {
  ...data,
  user_list: selectedUsers.map(user =>  ({ username: user })
}
```